components/button/stories/button.stories.js

```diff
@@ -349,4 +349,13 @@ Wrapping.args = {
 	showIconOnlyButton: false,
 	variant: "accent",
 	label: "An example of text overflow behavior within the button component. When the button text is too long for the horizontal space available, it wraps to form another line.",
-};
\ No newline at end of file
+};
+
+export const DownStateDocs = CustomButton.bind({});
+DownStateDocs.args = {
+	variant: "accent",
+	customStyles: {
+		"--spectrum-downstate-width": "72px",
+		"--spectrum-downstate-height": "32px"
+	}
+}
```

Comment:
This is a WIP and I don't love the direction it's going so far...if the team agrees on this approach (to setup down state in each component's index.css), then we should do this in the template.js, and make it compatible with changing t-shirt sizing and platform scale. But this was the first step to just see it working on the new Foundations/Down state page.

Checkbox doesn't need this because it uses the `component-size-minimum-perspective-down` token, which is already defined. But for components that are larger than 24px, we have to [calculate the perspective](https://github.com/adobe/spectrum-css/pull/2520/files#diff-97b5a7e747486aca74b3076354c28a00ef69dab3c91dc0806d926078198571a2R152) based on width and height. We've thought from the beginning that consumers would have to create their own implementation for setting the values of those width and height variables, and in button's case I think that's still true, because we don't statically set a width and height in the CSS...